### ISSUE TITLE
Integrate AchillesWeb

### ISSUE THREAD

**Issue by bastiao** [bioinformatics-ua/catalogue#784]
https://github.com/OHDSI/AchillesWeb

**Comment by bastiao** [comment]
Related PR can be found: https://github.com/OHDSI/AchillesWeb/pull/44

**Comment by tnovo** [comment]
related commits
73208e338883686ba59f699875c1cd8374aafda3 
de79685b0004eda9e32813948ecfacbdb669aef6

**Comment by ribeiroct** [comment]
For reference, this is being developed on : https://github.com/bioinformatics-ua/catalogue/tree/new/achilles

--
@tnovo : 
I've been testing it on my computer so we can it next week to the GA, and i have a few repairs:

- When we open the achilles tab on the fingerprint, and there's no content, we should have a brief message indicated the missing content, or in case of being one of the fingerprint owners, showing the "Add" button, and a message saying its possible to update.
- The iframe when there's no content starts to small to be able to see the modal (a min height of at least 400px should be set since the iframeResizer can't get height from absolute positioned elements such as the modal).
- The "Save Changes" button doesn't work, currently the only way to update or add a data source, is through an "Enter" while having selected the input.
- When we save, the request should be made through ajax, and the plugin should automatically get the new content when it finishes, also through ajax, we shouldn't have to refresh the page to get the new content. Currently we get a json output to the screen saying sucess.
- There's a bug on api.py in the achilles app. references to the class should be made through self. At least on python 2.7.8 https://github.com/bioinformatics-ua/catalogue/blob/new/achilles/emif/achilles/api.py#L82 <- this didn't work it should be "self.isValidDatasource(ds.datasource_url)" and the def below should have the self as the first parameter. I had to change this for it to run on my computer.
- The navbar where the menu stands and the detail view (where charts show) don't have the same width, they should. I would also probably align the updated button to the right and add it an icon, but that is just personal taste.
- We should have a static achillesweb deploy to show off, and it should be versioned. It's okay the django project that you sent me to test, but I'm installing unecessary dependencies, creating a new virtualenv etc, and the only thing that it does is url rewriting. Its way easier to test it if we can just use a static web server like apache or nginx directly with url rewrite rules. To do it we just need the reports and datasources folder. I tried it and it works just the same.

All being said, great work. ;)
